src/toolbars/AudioSetupToolBar.h

```diff
@@ -0,0 +1,91 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  AudioSetupToolBar.h
```

Comment:
One commit adds this file (but does much else besides), another corrects the comment.

I prefer to see a squashed sequence of commits that doesn't leave a record of mistakes and corrections.

Reply:
Commit squashed.